ext/soap/php_encoding.c

```diff
@@ -830,33 +831,34 @@ static zval *to_zval_hexbin(zval *ret, encodeTypePtr type, xmlNodePtr data)
 	return ret;
 }
 
+static zend_string *get_serialization_string_from_zval(zval *data)
```

Comment:
Should this have a "SOAP" prefix? I don't remember if SOAP uses them or  not

Reply:
The other methods here don't do that, most methods in soap actually don't :shrug: